cloud/cloud_status.go

```diff
@@ -3,27 +3,32 @@ package cloud
 type CloudStatus int
 
 const (
-	//Catch-all for unrecognized status codes
+	// StatusUnknown is a catch-all for unrecognized status codes.
 	StatusUnknown = CloudStatus(iota)
 
-	//StatusPending indicates that it is not yet clear if the instance
-	//has been successfully started or not (e.g., pending spot request)
+	// StatusPending indicates that it is not yet clear if the instance has been
+	// successfully started or not (e.g. pending spot request).
 	StatusPending
 
-	//StatusInitializing means the instance request has been successfully
-	//fulfilled, but it's not yet done booting up
+	// StatusInitializing means the instance request has been successfully
+	// fulfilled, but it's not yet done booting up.
 	StatusInitializing
 
-	//StatusFailed indicates that an attempt to start the instance has failed;
-	//Could be due to billing, lack of capacity, etc.
+	// StatusFailed indicates that an attempt to start the instance has failed;
+	// Could be due to billing, lack of capacity, etc.
```

Comment:
`could` or change L17 to have a period instead of a semicolon.

Reply:
Fixed.